Tasks/VsTest/vstest.ts

```diff
@@ -28,6 +29,7 @@ const systemDefaultWorkingDirectory = tl.getVariable('System.DefaultWorkingDirec
 const workingDirectory = systemDefaultWorkingDirectory;
 let testAssemblyFiles = undefined;
 let resultsDirectory = null;
+const taskProps = { errorCode: '', errorMessage: ''};
```

Comment:
use "result" for publishing the error message "resultCode" for code. we are already using this property